Specs/Scene/MaterialSpec.js

```diff
@@ -489,8 +497,9 @@ defineSuite([
             color : new Color(0.0, 1.0, 0.0, 1.0)
         });
 
-        var pixel = renderMaterial(material1);
-        expect(pixel).toEqual([0, 255, 0, 255]);
+        renderMaterial(material1, false, function(rgba) {
+            expect(rgba).toEqual([0, 255, 0, 255]);
+        });
```

Comment:
All these callbacks just check the color, could the expected color be sent to `renderMaterial` instead?